src/main/java/uk/gov/pay/adminusers/service/NewUserExistingServiceInviteCompleter.java

```diff
@@ -49,8 +51,12 @@ public InviteCompleteResponse complete(InviteEntity inviteEntity) {
         if (!inviteEntity.isUserType()) {
             throw internalServerError(format("Attempting to complete a 'new user, existing service' invite for an non user invite. invite-code = %s", inviteEntity.getCode()));
         }
+        if (secondFactor == null) {
+            throw internalServerError(format("Second factor method not provided when attempting to complete a 'new user, existing service' invite. invite-code = %s", inviteEntity.getCode()));
```

Comment:
I think this error should be a bad request error, and return a 400 status code because it's failing because something is missing from the request body, rather than anything internal to the app that's gone wrong.